config/crd/bases/replication.storage.openshift.io_volumereplications.yaml

```diff
@@ -179,6 +179,12 @@ spec:
               lastStartTime:
                 format: date-time
                 type: string
+              lastSyncBytes:
+                format: int64
+                type: integer
+              lastSyncDuration:
+                format: date-time
```

Comment:
is this really date-time? is there not a `duration` format, there is no date in this, I think?